elfi/methods/parameter_inference.py

```diff
@@ -834,8 +835,7 @@ def __init__(self,
         super(BayesianOptimization, self).__init__(
             model, output_names, batch_size=batch_size, **kwargs)
 
-        target_model = target_model or \
-            GPyRegression(self.model.parameter_names, bounds=bounds)
+        target_model = GPflowRegression(self.model.parameter_names, bounds=bounds)
```

Comment:
The `target_model` must be user-givable (as it was), though the default case can be discussed. :)

Reply:
Yes, this was a quick fix on my part: I'm not trying to dethrone GPy from the default.